protos/SignalService.proto

```diff
@@ -12,6 +12,7 @@ message Envelope {
     PREKEY_BUNDLE = 3; //Used By Signal. DO NOT TOUCH! we don't use this at all.
     RECEIPT       = 5;
     UNIDENTIFIED_SENDER = 6;
+    MEDIUM_GROUP_CIPHERTEXT = 7;
```

Comment:
My understanding was that we added stuff of our own after 100 in the case we have to pull some  stuff from signal. `MEDIUM_GROUP_CIPHERTEXT` is considered a signal stuff?

Reply:
I deliberately chose not to use 100+ values, because I don't think we should compromise our codebase because of Signal. If Signal ever adds a new entry for 7 (and we ever pull changes), we'll just give it the next available value.